### DISCUSSION TITLE
Right Side Transparent Bar [WIP]

### DISCUSSION THREAD

**Discussion by SerraPi** [JakeStanger/ironbar#560] [Show and tell]
![2024-04-27-040756_hyprshot](https://github.com/JakeStanger/ironbar/assets/95191556/b255e3cf-028a-48d2-ae26-5310efba8fc8)
![2024-04-27-041112_hyprshot](https://github.com/JakeStanger/ironbar/assets/95191556/3a9772e4-ded0-4d63-9ea9-710674adc879)

**Comment by crimsonduelist** [comment]
@SerraPi 
how did you add the taskbar

**Reply by SerraPi** [reply]
Iirc there's a center/middle? ( Not sure of the exact name) Element that can be added in config between Left/Right (top/bottom).

Let me know if this works, if not I can take a look when I get home.

**Reply by crimsonduelist** [reply]
Is it a built in module you mean or u added another bar?
I thought ironbar had smthing built in for it but can't seem to find smthing in docs
U got a dotfiles repo?

**Reply by crimsonduelist** [reply]
F sry for the bother I think I found it in another of the config flexs apparently ironbar calls it launcher that s why I couldn't find it, was searching for taskbar or dock. I ll check it when I get back.
Would love to see dotfiles if u got them BTW cuz that looks kinda awesome! Tnx anyway!
